docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -922,6 +922,10 @@ instructions, in order to activate the `Macvtap` feature gate (case sensitive).
 #### Limitations
 
 - Live migration is not seamless, see [issue #5912](https://github.com/kubevirt/kubevirt/issues/5912#issuecomment-888938920)
+- On [KinD](https://github.com/kubernetes-sigs/kind) clusters, the user needs to
```

Comment:
This does not seem to be a limitation of the macvtap binding, it is a limitation of using it in a lab with kind.

Can you add it as a note warning? Although we place to remove this text in the near future, so I do not mind much on what is written here.

Reply:
Done.